manager/addvertex.go

```diff
@@ -102,7 +102,15 @@ func (v *addVertexQueryManager) AddVertex(label string, properties ...interface{
 	query := traversal.NewTraversal().AddV(label)
 
 	for i := 0; i < len(properties); i += 2 {
-		query.AddStep("property", properties[i], properties[i+1])
+		k := properties[i]
+		v := properties[i+1]
+		switch v.(type) {
+		case []interface{}:
```

Comment:
Would this catch all cases of lists? In the event that the calling function passes something like `[]int`, this would not be executed.

Reply:
will there ever be a case like this?